### ISSUE TITLE
Select to zoom?

### ISSUE THREAD

**Issue by matthieu-beteille** [reactivemarkets/react-financial-charts#371]
Hey!

I have been looking through the code/examples - I couldn't find a simple example of a "zoom" feature where you can select a sub-part of the chart by click-and-dragging on the chart?

Is that not a feature? or did I just miss it?

**Comment by markmcdowell** [comment]
Hey, no unfortunately the original library we forked from didn't support it. I'm due to start a rewrite from next week so I can add it to the list of features.

**Comment by matthieu-beteille** [comment]
Thanks for the quick reply. Yea that would be great thanks. Are you planning to improve the typescript types as part of the rewrite as well?

Sounds like I am asking more features, so I feel like I should say that I love this charting lib it's very cool!

**Comment by markmcdowell** [comment]
Yes it'll be a full rewrite in typescript without any of the small hacks I did with `any`/funcs.

**Comment by matthieu-beteille** [comment]
That's very exciting, I am looking forward to it! Thanks.

**Comment by markmcdowell** [comment]
Not stale.

**Comment by Mograbi** [comment]
Hey, is this feature been resolved or not yet ? 

Thanks
